src/content/docs/en/concepts/islands.mdx

```diff
@@ -84,4 +84,45 @@ Even better, you can tell Astro exactly how and when to render each component. I
 
 In Astro, it’s up to you as the developer to explicitly tell Astro which components on the page need to also run in the browser. Astro will only hydrate exactly what’s needed on the page and leave the rest of your site as static HTML. 
 
-**Islands are the secret to Astro’s fast-by-default performance story!**
+**Client islands are the secret to Astro’s fast-by-default performance story!**
+
+## Server islands
+
+Each server island is a UI component that is rendered to the page independently from the rest, making it easy to combine high performance static HTML and dynamic server-generated components.
+
+Add the [`server:defer` directive](/en/reference/directives-reference/#server-directives) to any component on your page to turn it into its own island:
+
+```astro title="src/pages/index.astro" "server:defer"
+---
+import Avatar from '../components/Avatar.astro';
+---
+<Avatar server:defer />
+```
+
+This allows your page's critical content to be rendered immediately as a shell, while deferring your most dynamic UI elements, such as user avatars or product reviews. Different API endpoints load at different speeds, and server islands all load individually in parallel. A slower island won’t delay the rest of the page's personalized content from being available.
+
+Server islands provide a single caching solution for pages that combine multiple types of content:
+
+- Completely static and never changes.
+- Dynamically backed by a database that changes infrequently, but more often than you deploy.
+- Personalized content, tailored to individual users.
+
+Astro's server island implementation happens mostly at build time, where component content is swapped out for a small script. When Astro builds your site, it will omit the component and inject a script in its place, with fallback content will be visible for a short amount of time before the dynamic islands are then loaded.
+
+This rendering pattern was built to be portable. It does not depend on any server infrastructure so it will work with any host you have, from a Node.js server in a Docker container, to the serverless provider of your choice.
+
+### Benefits of server islands
+
+The most obvious benefit of server islands is the ability to render the more highly dynamic parts of your page on the fly. This allows the outer shell and main content to be more aggressively cached, providing faster performance.
```

Comment:
```suggestion
Server islands is the ability to render the more highly dynamic parts of your page on the fly, leaving the outer shell and main content to be more aggressively cached, providing faster performance.
```

Reply:
So the push back I'd give here is that we don't really need to redefine server islands at this point. This section is about the benefits, and it should probably start like the client island section does, which is exactly this "The most obvious benefit".

Instead of removing that, if you had to fill in the blank to finish that (or a similar sentence), what would you say?